usaspending_api/accounts/serializers.py

```diff
@@ -122,3 +122,11 @@ class ObjectClassFinancialSpendingSerializer(serializers.Serializer):
     major_object_class_code = serializers.CharField()
     major_object_class_name = serializers.CharField()
     obligated_amount = serializers.DecimalField(21, 2)
+
+
+class AgenciesFinancialBalancesSerializer(serializers.Serializer):
+
+    fiscal_year = serializers.IntegerField()
```

Comment:
Note--these are mine, so you can ignore them...I have a hotfix in to set these to `None`